### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by misaochan**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     android:versionName="1.17" >
 
     <uses-sdk
-        android:minSdkVersion="9"
+        android:minSdkVersion="11"
```
Comment (nicolas-raoul) [inline app/src/main/AndroidManifest.xml]: Android 3.0 if I am correct. Why is this change needed?

Comment (misaochan) [inline app/src/main/AndroidManifest.xml]: The Gradle build file says `minSdkVersion 11`, and the value specified in the Gradle build file overrides the manifest. Actually my IDE suggests removing the target SDK and min SDK lines entirely from the manifest, since the Gradle build file takes precedence so they are irrelevant. But I figured that preventing confusion by making the manifest values match the Gradle values would be sufficient.

Comment (nicolas-raoul) [inline app/src/main/AndroidManifest.xml]: I see, thanks!
Would you mind removing that from the SDK then? Better not be redundant :-)

**Comment by misaochan** [comment]
Okay, done
